types/react-tracking/index.d.ts

```diff
@@ -55,15 +56,26 @@ export type TrackingInfo<T, P, S> = T | ((props: P, state: S, args: any[any]) =>
 // Duplicated from ES6 lib to remove the `void` typing, otherwise `track` can’t be used as a HOC function that passes
 // through a JSX component that be used without casting.
 type ClassDecorator = <TFunction extends Function>(target: TFunction) => TFunction;
-type MethodDecorator = <T>(target: object, propertyKey: string | symbol, descriptor: TypedPropertyDescriptor<T>) => TypedPropertyDescriptor<T>;
+type MethodDecorator = <T>(
+    target: object,
+    propertyKey: string | symbol,
+    descriptor: TypedPropertyDescriptor<T>
+) => TypedPropertyDescriptor<T>;
 export type Decorator = ClassDecorator & MethodDecorator;
 
+/**
+ * A React context used to support passing and dispatching tracking data throughout a tree of components.
+ */
 export type TrackingContext<T = any> = React.Context<{
-    tracking: Options<T> & { data?: {} }
+    tracking: Options<T> & { data?: {} };
 }>;
-
 export const ReactTrackingContext: TrackingContext;
 
+/**
+ * A React hook used to tap into the tracking context.
+ */
+export function useTracking<P = {}>(): TrackingProp<P>;
```

Comment:
This doesn’t allow typing the function without using it, which means we’d need a wrapper function just to restrict it by default to our schema, but we can follow up with that in a later PR.